src/index.html

```diff
@@ -1,13 +1,124 @@
+            <img
+              class="header__tickets-img"
+              src="/images/icons/tickets-go.svg" alt="Tickets Go">
+          </a>
+        </div>
+
+        <div class="header__events">
+          <p class="header__events-text">події</p>
+        </div>
+      </div>
+
+      <div class="header__picture">
+        <a href="/" class="header__picture-logo">NA<br>MU</a>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="top-actions menu__top">
+            <a href="/" class="logo">
+
+            </a>
+          </div>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section page__section--first" id="current-events">
+        <div class="container">
+          <div class="current-events__content">
+            <div class="current-events__title">
+              <p class="current-events__text">Актуальні події</p>
+
+              <div class="current-events__image">
+                <img
+                  class="current-events__image-pic"
+                  src="/images/current-events.jpg"
+                  alt="current-events">
+              </div>
+
+              <div class="current-events__description">
+
+                <div class="current-events__date">
+                  <p class="current-events__text-grey">виставка</p>
+                  <p class="current-events__text-date">26.08-29.11.2019</p>
+                </div>
+
+                <div class="current-events__title">Йду i повертаюсь</div>
+
+                <div class="current-events__text-description">Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво другої половини 1980-1995 років.</div>
+              </div>
+            </div>
+
+            <div class="current-events__image">
+              <img
+                class="current-events__image-pic"
+                src="/images/current-events2.jpg"
+                alt="current-events">
+            </div>
+
+            <div class="current-events__description">
+
+              <div class="current-events__date">
+                <p class="current-events__text-grey">виставка</p>
+                <p class="current-events__text-date">26.08-29.11.2019</p>
+              </div>
+
+              <div class="current-events__title">I спогади i мрії</div>
+
+              <div class="current-events__text-description">Національний художній музей України до 100 річчя від дня народження   видатної української художниці  Тетяни Яблонської  відкриває ретроспективну ювілейну виставку «І спогади i мрії».</div>
+            </div>
+
+            <div class="current-events__bottom">
+
+              <div class="current-events__bottom--image">
+                <img
+                  class="current-events__bottom--image-pic"
+                  src="/images/bottom.jpg"
+                  alt="current-events">
+              </div>
+
```

Comment:
Try to rewrite without `img`, use background-img, could use as e[ample footer from Maimi, watch [this video](https://mate.academy/learn/css-advanced/css-advanced-landing#/video/1308)